react/features/base/connection/actions.web.js

```diff
@@ -22,7 +23,7 @@ import logger from './logger';
 export function connect() {
     return (dispatch: Dispatch<any>, getState: Function) => {
         // XXX Lib-jitsi-meet does not accept uppercase letters.
-        const room = getState()['features/base/conference'].room.toLowerCase();
+        const room = getBackendSafeRoomName(getState()['features/base/conference'].room);
```

Comment:
Looks like we are calling `getBackendSafeRoomName` on the value stored in redux in all occasions. This makies me think that Teh Right Thing To Do Here (TM) would be to store it in its proper form. The reducer would do this in the SET_ROOM action handler (IIRC). This follows the "normalize on the way in, use the nbormaliozed value henceforth" pattern, which I think will help us avoid future bugs if the room value is not normalized, WDYT?

Reply:
We discussed this offline. If we change the value in the store, then we need to refactor other places where a display-capable version of the room name is needed, for example in the navbar on mobile, or the recent list, or invite link generation...etc